src/main/java/me/eccentric_nz/tardisvortexmanipulator/command/TvmCommand.java

```diff
@@ -66,7 +67,7 @@ public boolean onCommand(@NotNull CommandSender sender, Command command, @NotNul
                 return true;
             }
             ItemStack itemStack = player.getInventory().getItemInMainHand();
-            if (itemStack.hasItemMeta() && itemStack.getItemMeta().hasDisplayName() && itemStack.getItemMeta().getDisplayName().equals("Vortex Manipulator")) {
+            if (itemStack.hasItemMeta() && itemStack.getItemMeta().getPersistentDataContainer().get(TardisVortexManipulatorPlugin.plugin.getItemKey(), PersistentDataType.STRING).equals("vortex_manipulator")) {
```

Comment:
You just broke every Vortex Manipulator created before adding this...

Reply:
yeah, so maybe i can figure out a way to add PBVs to all legacy ones